src/components/timeseries.js

```diff
@@ -495,6 +503,32 @@ function TimeSeries(props) {
           />
         </div>
       </div>
+
+      <div className="pills" style={{marginTop: '32px', textAlign: 'right'}}>
```

Comment:
The SVG above these controls overflows and pointer events are intercepted, leading to these buttons not receiving click-events. Gave `margin-top` to work around this issue.